src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -60,6 +60,7 @@ module sonic-types {
 
     typedef packet_action{
         type enumeration {
+            enum ACCEPT;
```

Comment:
Is it only used in control plane ACL? If yes, should we limit the enum used by data plane ACL?

Reply:
Didn't find any doc talking more about PACKET_ACTION.
[ACL Design](https://github.com/Azure/SONiC/blob/master/doc/acl/ACL-High-Level-Design.mdl) mentioned "FORWARD and DROP" for "L3" tables